packages/expo-updates/android/src/main/java/expo/modules/updates/loader/EmbeddedLoader.kt

```diff
@@ -22,11 +22,6 @@ import java.util.*
  * update. The benefits of this include (a) a single code path for launching most updates and (b)
  * assets included in embedded updates and copied into the cache in this way do not need to be
  * redownloaded if included in future updates.
- *
- * However, if a visual asset is included at multiple scales in an embedded update, we don't have
- * access to and must skip copying scales that don't match the resolution of the current device. In
- * this case, we cannot fully copy the embedded update, and instead launch it from the original
- * location. We still copy the assets we can so they don't need to be redownloaded in the future.
  */
```

Comment:
cc @ide / @lukmccall - if you happen to have any insight into the behavior described in this comment block, it'd be helpful. @douglowder hasn't been able to reproduce any issues here

Reply:
The issue described in the comment isn't entirely surprising to me: it is possible that app stores create builds with only a subset of assets. For instance, an arm64 iOS build might only run on a device with 2x resolution (would need to look at all the iPhone SKUs) and could exclude 1x assets.

However, that's just a guess. And app stores need to anticipate device upgrades: what happens when a user makes a backup on a 2x device and restores it on a new 3x device?

Reply:
Actually the file that performs pixel density check in a cotlin file and thus seems like an Android thing. I could not find a similar check in Swift files

Reply:
That is correct @prateekbh this is an Android specific issue.

Reply:
>The issue described in the comment isn't entirely surprising to me: it is possible that app stores create builds with only a subset of assets.

That might be possible but we have a gating logic to pick a subset from what's bundled with the app. Can we just copy whatever the app carries?

Reply:
@prateekbh this PR should do exactly that -- copy everything in the app.

It would probably be worth it to try a build with this change, and test your app (since it has a large number of assets) on low-end Android devices to make sure that there are no issues.